### PR TITLE
fix README dependencies link in PR body

### PR DESCRIPTION
**Pull request by pshergie**
No issue

# Motivation

wanted to improve DX

# Screenshot or Video

pretty minor change for a video

# Checklist

- [x] Don’t rush. Check all changes in PR again.
- [x] Run `pnpm test`.
- [x] Think about changing documentation.
  - If you added a script to `scripts/`, add a comment with a description.
  - If you added a new folder, add its description to the project’s `README.md`.
  - If you added config, describe how we use this tool in the config’s comment.
  - If you added something to the project’s architecture, describe it in the project’s `README.md`.
  - Try to focus on “why?”, not “how?”.
- [ ] If you added a new dependency, check our [requirements](https://github.com/hplush/slowreader/blob/main/README.md#dependencies).
- [ ] Think about testing
  - If you added a feature, add unit tests.
  - If you added a new state to the UI, add visual tests.
  - If you fixed the bug, think about preventing bug regression in the future.
- If you changed web client:
  - [ ] Think about moving code to `core/`. What code will also be useful on other platforms?
  - [ ] Run `pnpm size` and check the difference in the JS bundle size. Is it relevant to the changes? Change the limit in `web/.size-limit.json` if necessary.
  - [ ] The UI was checked in Chrome and Firefox (and Safari or Epiphany if you have them).
  - [ ] Think about keyboard UX. Is it easy to use the new feature with only one hand on a keyboard? Is it easy to understand what keys to press?
  - [ ] Think about HTML semantics.
  - [ ] Think about accessibility. Check a11y recommendations. Think about how screen reader users will use the tool. Is it easy to use on a screen with bad contrast?
  - [ ] Think about translations.
  - [ ] Think about right-to-left languages. What parts of the screen should be mirrored for Arabic or Hebrew languages?
- If you changed `web/main/` files:
  - Do you need to backport styles changes to `web/public/404.html` or `web/public/500.html`?
  - Think about app loading styles inlined in `index.html`.
- If you changed `core/`:
  - [ ] Think about making types more precise. Can you better explain data relations by type?
  - [ ] Think about conflict resolution. We don’t need some very smart changing merging; just 2 changes of the same item on different clients should not break the database. What if the user changes an item on one machine and removes it on another?
  - [ ] Think about log and storage migration.
- If you changed English translations:
  - [ ] Change translation ID if you change the meaning of the text.

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Fixes # (PUT ISSUE NUMBER AFTER #)
   - If you added config, describe how we use this tool in the config’s comment.
   - If you added something to the project’s architecture, describe it in the project’s `README.md`.
   - Try to focus on “why?”, not “how?”.
-- [ ] If you added a new dependency, check our [requirements](../README.md#dependencies).
+- [ ] If you added a new dependency, check our [requirements](../blob/main/README.md#dependencies).
```
Comment (ai) [inline docs/pull_request_template.md:23]: Try a link in your fork https://github.com/pshergie/slowreader/blob/fix-pr-body-checklist-link/docs/pull_request_template.md

It doesn’t work right now (404).

What exact issue do you have right now?

Comment (pshergie) [reply docs/pull_request_template.md:23]: the issue: the `requirements` link doesn't work from PRs

Comment (ai) [reply docs/pull_request_template.md:23]: In this case try to avoid `../` in the link, give a direct one `https://`

Comment (pshergie) [reply docs/pull_request_template.md:23]: looks like `[a relative link](other_file.md)` could work https://github.blog/2013-01-31-relative-links-in-markup-files/

Comment (ai) [reply docs/pull_request_template.md:23]: Sometimes docs are old. That one from 2013.

You see that link doesn’t work if you open `.md` file in GitHub.

Comment (pshergie) [reply docs/pull_request_template.md]: @ai ready for the review 🙇

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Fixes # (PUT ISSUE NUMBER AFTER #)
   - If you added config, describe how we use this tool in the config’s comment.
   - If you added something to the project’s architecture, describe it in the project’s `README.md`.
   - Try to focus on “why?”, not “how?”.
-- [ ] If you added a new dependency, check our [requirements](../README.md#dependencies).
+- [ ] If you added a new dependency, check our [requirements](/hplush/slowreader/blob/main/README.md#dependencies).
```
Comment (ai) [inline docs/pull_request_template.md:23]: Try to open this link from https://github.com/pshergie/slowreader/blob/fix-pr-body-checklist-link/docs/pull_request_template.md

Comment (pshergie) [reply docs/pull_request_template.md]: The case you described is not common as the file is intended to be interacted with through the pull request page but I fixed it though. Let's hope github doesn't change its URL soon

**Comment by ai** [comment]
Thanks. Now the link works.

**Pull request merged**